pkg/cmd/indices/config/config.go

```diff
@@ -0,0 +1,22 @@
+package config
+
+import (
+	"github.com/spf13/cobra"
+
+	indiceexport "github.com/algolia/cli/pkg/cmd/indices/config/export"
```

Comment:
Could we rename those two to `configexport` ?

Reply:
Yes! 👍